docs/devel/api_changes.md

```diff
@@ -485,6 +485,17 @@ To regenerate them:
 hack/update-generated-deep-copies.sh
 ```
 
+Since our components still use internal apis, you may need to generate code
```

Comment:
@ddysher - sorry for late response - I was travelling to US yesterday, and preparing for US visit previously :)
https://github.com/kubernetes/kubernetes/pull/23427

is the first PR migrating to the new generator
There will 3-4 more subsequent PRs (hopefully within next week).

I will get back to this PR once this migration is finished. Does it sound ok for you?